graderx/graders/stdout_graders/stdout_common/stdout_common.py

```diff
@@ -1,29 +1,58 @@
 from pathlib import Path
 import json
 from .lib import test_cases_parser as tc_parser
+import shutil
 
 
 def add_test_case_file(path, tc_name, data):
     path = Path(path, tc_name)
     with path.open('w') as write_file:
         write_file.write(data)
 
+def create_course_dir(course_name):
+    course_path = Path(__file__).joinpath(f"../../../courses/{course_name}").resolve()
+    course_path.mkdir(parents=True, exist_ok=True)
+
+def create_lab_dir(course, lab):
+    lab_path = Path(__file__).joinpath(f"../../../courses/{course}/labs/{lab}").resolve()
+    lab_path.mkdir(parents=True, exist_ok=True)
+
+
 def create_course_data(course_name, labs):
     labs_path = Path(__file__).joinpath(f"../../../courses/{course_name}/labs").resolve()
     for lab in labs:
         test_cases_path = labs_path.joinpath(f"{lab['name']}/test_cases")
         test_cases_path.mkdir(parents=True, exist_ok=True)
-        create_test_cases(test_cases_path, lab['test_cases'])
 
-def create_test_cases(path, test_cases):
+def create_test_cases(course, lab, test_cases):
+    test_cases_path = Path(__file__).joinpath(f"../../../courses/{course}/labs/{lab}/test_cases").resolve()
+    test_cases_path.mkdir(parents=True, exist_ok=True)
     for tc in test_cases:
-        add_test_case_file(path, f"{tc['id']}_in", tc['input'])
-        add_test_case_file(path, f"{tc['id']}_out", tc['output'])
+        add_test_case_file(test_cases_path, f"{tc['id']}_in", tc['input'])
+        add_test_case_file(test_cases_path, f"{tc['id']}_out", tc['output'])
+
+def clear_test_cases(course, lab):
+    test_cases_path = Path(__file__).joinpath(f"../../../courses/{course}/labs/{lab}/test_cases").resolve()
+    if test_cases_path.exists():
```

Comment:
It's better to refactor `../../../courses` to a constant, then use `os.path.join` function to join paths, having a long string in the code makes it harder to read